setup.py

```diff
@@ -44,10 +44,6 @@ def _discover_tests():
         long_description=LONG_DESCRIPTION,
         license='GPL-3.0',
         packages=find_packages(),
-        package_data={
-            'orangecontrib.geo.widgets': ['icons/*',
```

Comment:
I'm not an expert in setup files. Why can "icons/*" be removed from this list?

Reply:
Because they are already in manifest.in.